maxide.bmx

```diff
@@ -7767,6 +7827,25 @@ Type TCodePlay
 
 		Return Null
 	End Method
+	
+	
+	Method CanRunUPX:int()
+		local platform:string = GetPlatform()
+		If platform = "emscripten" Or platform = "nx" Or platform = "ios" Then
+			Return False
+		End If
+		
+		Local upx:String = BlitzMaxPath() + "/bin/upx"
+		?win32
+		upx :+ ".exe"
+		?
+		If FileType(upx) = 1

```

Comment:
You shouldn't use seemingly random numbers when there are consts available. (e.g. FILETYPE_FILE)

When I change file types to an enum, all your code will break...

Reply:
Will change in the next commit.